deployments/production/spark-worker-daemonset.json

```diff
@@ -41,7 +39,19 @@
                             },
```

Comment:
ditto other comments - we'll need to update this if dns is resolving differently

Reply:
Service name (dns hostname) has not changed (see my other comment).